ingest-app/src/cmr/ingest/api/variables.clj

```diff
@@ -28,13 +28,21 @@
   [headers]
   (mt/extract-header-mime-type #{mt/json} headers "content-type" true))
 
+(defn- snake-case-data
```

Comment:
That's probably a good idea and it seems worthy of unit tests with different data structures.